migration/i8.2z/oracle/202106161301_IDEMPIERE-4837.sql

```diff
@@ -0,0 +1,22 @@
+SET SQLBLANKLINES ON
+SET DEFINE OFF
+
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
```

Comment:
Please:
* move the scripts to folder i8.2
* in the case of windows the predefined context must be added to ad_window instead of ad_menu, otherwise the context will be added just when opened from menu, not when zooming
* set the comment for the script instead of "I forgot to ..."
* I think the variable must be the opposite "UsePriceListInProductInfo" (instead of Ignore) ... see my comment below, so, the list of windows to update will be different.